### DISCUSSION TITLE
[Question]: Do plugins only work with gpt-X models?

### DISCUSSION THREAD

**Discussion by ncecere** [danny-avila/LibreChat#2810] [Q&A] [answered]
### What is your question?

Do the plugin only work with OpenAI GPT models?

### More Details

I am running a Litellm server and hosting several models behind it.  It is providing we an OpenAI compatible API.  When I add plugin store and  specify to use a model like llama3-8b-instruct and set the OpenAI API reverse proxy I never get an image.  I will get a description but the image never shows up.  Am i doing something wrong is is this the expected behavior ?  Is this also true for all models and plugins?

### Screenshots

<img width="1040" alt="image" src="https://github.com/danny-avila/LibreChat/assets/6109463/a27ed44e-f92a-49a8-832c-2482d84c76a5">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
Yes, see the discussions here: https://github.com/danny-avila/LibreChat/discussions/2793 https://github.com/danny-avila/LibreChat/discussions/2557

**Reply by ncecere** [reply]
thank you for the reply, this did answer my question.
